src/rtc/mosaic_geobursts.py

```diff
@@ -599,7 +617,8 @@ def mosaic_single_output_file(list_rtc_images, list_nlooks, mosaic_filename,
     raster_out = drv_out.Create(mosaic_filename, width, length, num_bands,
                                 datatype_mosaic)
 
-    raster_out.SetGeoTransform((xmin_mosaic, posting_x, 0, ymax_mosaic, 0, posting_y))
+    raster_out.SetGeoTransform((xmin_mosaic, posting_x, 0, ymax_mosaic, 0,
+                                posting_y))
```

Comment:
```suggestion
    raster_out.SetGeoTransform((xmin_mosaic, posting_x, 0,
                                ymax_mosaic, 0, posting_y))
```
Just wanted to improve the symmetry of the lines for the parameters, which can help the readability. Up to you whether or not to take the suggestion.